Code/Engine/RendererVulkan/RendererVulkanDLL.h

```diff
@@ -5,10 +5,14 @@
 
 // Configure the DLL Import/Export Define
 #if EZ_ENABLED(EZ_COMPILE_ENGINE_AS_DLL)
-#  ifdef BUILDSYSTEM_BUILDING_RENDERERVULKAN_LIB
-#    define EZ_RENDERERVULKAN_DLL __declspec(dllexport)
+#  if EZ_ENABLED(EZ_PLATFORM_WINDOWS)
+#    ifdef BUILDSYSTEM_BUILDING_RENDERERVULKAN_LIB
+#      define EZ_RENDERERVULKAN_DLL __declspec(dllexport)
+#    else
+#      define EZ_RENDERERVULKAN_DLL __declspec(dllimport)
+#    endif
 #  else
-#    define EZ_RENDERERVULKAN_DLL __declspec(dllimport)
+#    define EZ_RENDERERVULKAN_DLL __attribute__((visibility("default")))
```

Comment:
Maybe we should add a macro in Foundation that wraps __declspec(dllimport) and __attribute__(...), so that we don't need to do all that in every lib.

Reply:
Like EZ_DECL_EXPORT and EZ_DECL_IMPORT